app/tests/fixtures/dynamodb.py

```diff
@@ -39,6 +45,14 @@ def setup_dynamodb_table() -> None:
         print(e)
 
 
+def delete_dynamodb_table() -> None:
+    try:
+        dynamodb = boto3.client('dynamodb', endpoint_url=f'http://localhost:{DB_PORT}')
+        dynamodb.delete_table(TableName=TABLE_NAME)
+    except dynamodb.exceptions.ResourceNotFoundException as e:
```

Comment:
You are not using the caught exception
```suggestion
    except dynamodb.exceptions.ResourceNotFoundException:
```